src/diffusers/schedulers/scheduling_dpmsolver_singlestep.py

```diff
@@ -517,34 +518,43 @@ def convert_model_output(
                 x0_pred = self._threshold_sample(x0_pred)
 
             return x0_pred
+
         # DPM-Solver needs to solve an integral of the noise prediction model.
-        elif self.config.algorithm_type == "dpmsolver":
+        elif self.config.algorithm_type in ["dpmsolver", "sde-dpmsolver"]:
```

Comment:
should we not support `"sde_dpmsolver" here if it does not work?

Reply:
@LuChengTHU says `sde_dpmsolver` is hard to use in practice and you are unwilling to support it; thus I will remove it.

Reply:
ok let's remove it then
we can merge this after that:)